coda/coda_validate/tests/test_views.py

```diff
@@ -369,3 +369,27 @@ def test_invalid_method_and_identifier_returns_bad_request(self, rf):
         request = rf.put('/', HTTP_HOST='example.com')
         response = views.app_validate(request)
         assert response.status_code == 400
+
+
+class TestCheckJson(object):
+
+    def test_with_no_statuses(self, rf):
+        request = rf.get('/')
+        response = views.check_json(request)
+        assert response.status_code == 200
+
+        counts = json.loads(response.content)
+        assert counts['Passed'] == 0
+        assert counts['Failed'] == 0
+        assert counts['Unverified'] == 0
+
+    def test_with_statuses(self, rf):
+        factories.ValidateFactory.create_batch(100)
+        request = rf.get('/')
+        response = views.check_json(request)
+        assert response.status_code == 200
+
+        counts = json.loads(response.content)
+        assert counts['Passed'] > 0
+        assert counts['Failed'] > 0
+        assert counts['Unverified'] > 0
```

Comment:
No, there is the very small possibility that a `Validate` object of with one of the three statuses will not be created. Assuming my math is correct the chance of that scenario occurring given is `1.9403252e-48` :smile: